modules/fedimint-ln-client/src/incoming.rs

```diff
@@ -159,6 +159,7 @@ impl FundingOfferState {
     ) -> Result<(), IncomingSmError> {
         debug!("Awaiting funding success for outpoint: {out_point:?}");
         for retry in 0.. {
+            let sleep = retry * 15;
```

Comment:
Perhaps enforce a maximum value here?
Something like:
```suggestion
            let sleep = max(retry * 15, 90);
```